src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml

```diff
@@ -1,7 +1,14 @@
 @{
     string type = Model.Type.ToString().ToLowerInvariant();
+    var bsClassName = type switch
+    {
+        "information" => "alert-info",
+        "error" => "alert-danger",
+        _ => $"alert-{type}",
+    };
 }
-<div class="alert alert-dismissible fade show message-@type alert-@type" role="alert">
```

Comment:
What's the wrong with the previous one `alert-@type` is fine for me

Reply:
Because alert-information does not work

Reply:
I see, is the enum values like this from the beginning?!!